core/src/test/scala/integration/kafka/api/QuotasTest.scala

```diff
@@ -15,12 +15,13 @@
 package kafka.api
```

Comment:
The link to the original failure that Dong posted isn't working any more, so I don't know what the root cause of the transient failure was - can you explain? In the mean time I will try to reproduce locally.